llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -237,16 +237,16 @@ def int_amdgcn_reloc_constant : DefaultAttrsIntrinsic<
 // the second one is copied to m0
 def int_amdgcn_s_sendmsg : ClangBuiltin<"__builtin_amdgcn_s_sendmsg">,
   Intrinsic <[], [llvm_i32_ty, llvm_i32_ty],
-  [ImmArg<ArgIndex<0>>, IntrNoMem, IntrHasSideEffects, IntrWillReturn]>;
+  [ImmArg<ArgIndex<0>>, IntrNoMem, IntrHasSideEffects, IntrWillReturn, IntrNoCallback]>;
```

Comment:
They are also `IntrNoRecurse` right?

Reply:
IntrNoRecurse was dropped from this version

Reply:
Do you mean that triggering the async execution of code in the module could also call this intrinsic, making it recursive?

Reply:
No, I mean the IntrNoRecurse no longer exists since I only added it for this case, which probably does not need it